ci-operator/config/openshift/ci-docs/openshift-ci-docs-master.yaml

```diff
@@ -42,7 +42,7 @@ resources:
       memory: 200Mi
 tests:
 - as: links
-  commands: htmlproofer --root-dir /src/public/ --check_sri --http-status-ignore 403,405
+  commands: htmlproofer --root-dir /src/public/ --check_sri --http-status-ignore 401,403,405
```

Comment:
To me it looks the `--url-ignore` list below would be a better option to solve the problem you see with `rover.redhat.com`? We want a specific URL to be fine, not to ignore all 401s..

Reply:
Updated.